org.eclipse.jdt.core/model/org/eclipse/jdt/internal/core/util/CommentRecorderParser.java

```diff
@@ -203,6 +203,11 @@ public int flushCommentsDefinedPriorTo(int position) {
 		return position;
 	}
 
+	@Override
+	protected void consumeExitTryBlock() {
+		flushCommentsDefinedPriorTo(this.scanner.currentPosition);
```

Comment:
@subyssurendran666 See some of the other overridden consume* methods in this class that call the equivalent methods on super after doing what they are doing. I think we should do the same here as well.

Reply:
Done